### ISSUE TITLE
problems with villagers head drops

### ISSUE THREAD

**Issue by chillerkraft** [Rocologo/MobHunting#254]
apparently, the librarian is a steve head, and the cleric doesnt drop a skull (i dont see it in config either)

**Comment by chillerkraft** [comment]
also, the bag of gold sign doesn't work anymore on my server. its still green letter but doesn't interact at all

**Comment by Rocologo** [comment]
I dont know what a cleric is???

and my guess is that librarian is spelled wrongly in your config.yml because this has been a bug in the past. But becuase I dont update your values in config.yml the bug has not been corrected in your config. check the spelling or delete the line "librarian-cmd" and restart your server and then the line will be auto created.

**Comment by chillerkraft** [comment]
apparently cleric is a type of villager.. but i guess not if you don't see it in the code.

i fixed librarian :)

**Comment by Rocologo** [comment]
Ahh now I know. Cleric is a Career. I only test for Profession. Which means that the reward for the Priest and the Cleric is the same. 

http://minecraft.gamepedia.com/Villager#Professions_and_careers

I only look at Profession because Villagers is not that important because they are not agreesive. I could easily test for career too, but then you would 13 new Mobs to configure, and because the are not that interesting too kill, I decided not to, when I learned about the difference between Profession and career.
